src/pathfinding/PathConditionReachable.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2018 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation,  either version 2 of the License,  or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not,  see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#ifndef PathConditionReachable_h__
+#define PathConditionReachable_h__
+
+#include "world/GameWorldBase.h"
+
+struct PathConditionReachable
+{
+    const GameWorldBase& world;
+
+	PathConditionReachable(const GameWorldBase& world) : world(world) {}
```

Comment:
Please clang-format. Tabs and new lines are wrong. Maybe you have to rerun cmake or better add an empty new line to CMakeLists.txt to trigger it. (We are using `file(GLOB` so new files don't trigger a re-run like it should)

Reply:
Clang format just died before reaching this file. The reason was that i forgot to re-run cmake after a renaming.